scripts/pull-manifests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+shopt -s extglob
+
+catalogd_version=$CATALOGD_VERSION
+cert_mgr_version=$CERT_MGR_VERSION
+rukpak_version=$RUKPAK_VERSION
+manifestsToClean=()
+
+# Let's check if the manifests for cert-manager, rukpak, and catalogd are available in the local filesystem
+# If they are, we'll use those instead of pulling from GitHub
+if [[ -f "testdata/manifests/cert-manager-${cert_mgr_version}.yaml" ]]; then
```

Comment:
Is there a better landing spot for these than `testdata`? What about `third_party` or `external` or `dependencies` or something like that? Or maybe something under `hack`, sort of as a sibling of `hack/tools`?

Reply:
I was thinking on something like `contrib` or `third_party` https://github.com/operator-framework/operator-controller/pull/233#discussion_r1205726677